### DISCUSSION TITLE
Display the absence of a person

### DISCUSSION THREAD

**Discussion by AurelienPillevesse** [nhn/tui.calendar#1295] [Q&A]
## Version
Tui React

## Test Environment
Windows - Chrome

## Current Behavior
Don't know how to display day off on the calendar.
I would like to know if it's possible for example, 15/09/2022, people see that i'm absent (not present) on my calendar ?

Best regards,

**Comment by adhrinae** [comment]
The calendar doesn't offer a such feature like that.
Perhaps you might need to create an arbitrary event that shows someone's absence.

Maybe related to #1227 because the author suggested a similar solution.

> I resolved it by using "createEvents" feature with customized css (expand to full width)
